docs/index.Rmd

```diff
@@ -605,16 +606,32 @@ document.addEventListener("DOMContentLoaded", function() {
     return (b !== "transparent" &&
             b !== "rgba(0, 0, 0, 0)" &&
             b !== "rgba(255,255,255,0)") ? b
-         : (el.parentNode === null)      ? "white"
+         : (el.tagName === "BODY")       ? "white"
                                          : getBGColor(el.parentNode);
   }
 
-  document.getElementById("button-go").addEventListener("click", function() {
+  const url = new URL(location);
```

Comment:
Let's implement without URL constructor, which is only available in modern browsers.